patterns/README.md

```diff
@@ -0,0 +1,8 @@
+# WooCommerce Blocks - Patterns
+
+## Naming Conventions for Patterns
+
+* Use spaces as a separator. Only use hyphens for compound words or expressions.
+* Use consistent title case capitalization.
+* Use clear, descriptive names that indicate the component's function or appearance.
+* Group similar items together with a common name structure."
```

Comment:
We're one typo away from merging!

```suggestion
* Group similar items together with a common name structure.
```

Reply:
@kmanijak lets do it then :) feel free to edit such stuff right away, no issue at all for me :)